src/social-share-button-qr.js

```
@@ -0,0 +1,176 @@
+/**
+ * SocialShareButton QR Code Extension
+ * Dynamically loads Kazuhiko Arase's qrcode-generator from CDN
+ */
+
+(function () {
+  function applyQRPatch() {
+    if (typeof window === "undefined" || !window.SocialShareButton) {
+      console.warn("SocialShareButton core must be loaded before QR extension");
```

ESLint

[error] Unexpected console statement.

(no-console)